### PR TITLE
translate 2022-08-29-csi-inline-volumes-ga.md

### PR DESCRIPTION
**Pull request by ldsdsy**

### CODE REVIEW

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+**Author:** Jonathan Dobson (Red Hat)
+-->
+**作者：** Jonathan Dobson (Red Hat)
+
+<!--
+CSI Inline Volumes were introduced as an alpha feature in Kubernetes 1.15 and have been beta since 1.16. We are happy to announce that this feature has graduated to General Availability (GA) status in Kubernetes 1.25.
+
+CSI Inline Volumes are similar to other ephemeral volume types, such as `configMap`, `downwardAPI` and `secret`. The important difference is that the storage is provided by a CSI driver, which allows the use of ephemeral storage provided by third-party vendors. The volume is defined as part of the pod spec and follows the lifecycle of the pod, meaning the volume is created once the pod is scheduled and destroyed when the pod is destroyed.
+-->
+CSI 内联存储卷是在 Kubernetes 1.15 中作为 alpha 功能推出的，并从 1.16 开始成为 beta 版本。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:24]:
```suggestion
CSI 内联存储卷是在 Kubernetes 1.15 中作为 Alpha 功能推出的，并从 1.16 开始成为 Beta 版本。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+metadata:
+  name: secrets-store.csi.k8s.io
+spec:
+  podInfoOnMount: true
+  attachRequired: false
+  volumeLifecycleModes:
+  - Ephemeral
+```
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:85]: Line 75-85 is not necessary to comment out if no trans.

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+        readOnly: true
+      command: [ "sleep", "1000000" ]
+  volumes:
+    - name: secrets-store-inline
+      csi:
+        driver: secrets-store.csi.k8s.io
+        readOnly: true
+        volumeAttributes:
+          secretProviderClass: "my-provider"
+```
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:110]: Same issue as above.

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+
+If the driver supports any volume attributes, you can provide these as part of the `spec` for the Pod as well:
+
+```
+      csi:
+        driver: block.csi.vendor.example
+        volumeAttributes:
+          foo: bar
+```
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:119]: Same issue as above.

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+
+Two existing CSI drivers that support the `Ephemeral` volume lifecycle mode are the Secrets Store CSI Driver and the Cert-Manager CSI Driver.
+
+The [Secrets Store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) allows users to mount secrets from external secret stores into a pod as an inline volume. This can be useful when the secrets are stored in an external managed service or Vault instance.
+
+The [Cert-Manager CSI Driver](https://github.com/cert-manager/csi-driver) works along with [cert-manager](https://cert-manager.io/) to seamlessly request and mount certificate key pairs into a pod. This allows the certificates to be renewed and updated in the application pod automatically.
+-->
+## 使用案例示例
+
+支持 `临时` 卷生命周期模式的两个现有 CSI 驱动是 Secrets Store CSI 驱动和 Cert-Manager CSI 驱动。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:183]:
```suggestion
支持`临时`卷生命周期模式的两个现有 CSI 驱动是 Secrets Store CSI 驱动和 Cert-Manager CSI 驱动。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+-->
+
+## 安全考虑因素
+
+应特别考虑哪些 CSI 驱动可作为内联卷使用。
+`volumeAttributes` 通常通过 `StorageClass` 控制，并可能包含应限制给集群管理员的属性。
+允许 CSI 驱动用于内联临时卷意味着任何有权限创建 Pod 的用户也可以通过 Pod spec 向驱动提供 `volumeAttributes` 字段。
+
+集群管理员可以选择从 CSIDriver spec 中的 `volumeLifecycleModes` 中省略（或删除） `Ephemeral`，
+以防止驱动被用作内联临时卷，或者使用[admission webhook](/docs/reference/access-authn-authz/extensible-admission-controllers/) 来限制驱动的使用。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:206]:
```suggestion
以防止驱动被用作内联临时卷，或者使用[准入 Webhook](/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers/) 来限制驱动的使用。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+- [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volumes)
+- [CSI documentation](https://kubernetes-csi.github.io/docs/ephemeral-local-volumes.html)
+- [KEP-596](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/596-csi-inline-volumes/README.md)
+- [Beta blog post for CSI Inline Volumes](https://kubernetes.io/blog/2020/01/21/csi-ephemeral-inline-volumes/)
+-->
+## 参考资料
+
+关于此功能的更多信息，请参阅：
+
+- [Kubernetes 文档](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volumes)
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:222]:
```suggestion
- [Kubernetes 文档](/zh-cn/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volumes)
```

**Review by windsonsea — COMMENTED** [review]
Overall looks fluently good to me. Thanks.

Do we need to keep consistent with the [localization glossary](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E6%9C%AF%E8%AF%AD%E5%AF%B9%E7%85%A7) for GA (General Availability)?

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 8]
```diff
@@ -0,0 +1,226 @@
...
+
+<!--
+CSI Inline Volumes were introduced as an alpha feature in Kubernetes 1.15 and have been beta since 1.16. We are happy to announce that this feature has graduated to General Availability (GA) status in Kubernetes 1.25.
+
+CSI Inline Volumes are similar to other ephemeral volume types, such as `configMap`, `downwardAPI` and `secret`. The important difference is that the storage is provided by a CSI driver, which allows the use of ephemeral storage provided by third-party vendors. The volume is defined as part of the pod spec and follows the lifecycle of the pod, meaning the volume is created once the pod is scheduled and destroyed when the pod is destroyed.
+-->
+CSI 内联存储卷是在 Kubernetes 1.15 中作为 alpha 功能推出的，并从 1.16 开始成为 beta 版本。
+我们很高兴地宣布，这项功能在 Kubernetes 1.25 版本中已经升级为通用（GA）状态。
+
+CSI 内联存储卷与其他临时卷类型相似，如  `configMap`、`downwardAPI` 和  `secret`。
```
Comment (Michelle951) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:27]:
```suggestion
CSI 内联存储卷与其类型的临时卷相似，如  `configMap`、`downwardAPI` 和  `secret`。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 9]
```diff
@@ -0,0 +1,226 @@
...
+
+There are a couple of new bug fixes related to this feature in 1.25, and the [CSIInlineVolume feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) has been locked to `True` with the graduation to GA. There are no new API changes, so users of this feature during beta should not notice any significant changes aside from these bug fixes.
+
+- [#89290 - CSI inline volumes should support fsGroup](https://github.com/kubernetes/kubernetes/issues/89290)
+- [#79980 - CSI volume reconstruction does not work for ephemeral volumes](https://github.com/kubernetes/kubernetes/issues/79980)
+-->
+
+## 1.25 版本有什么新内容？
+
+在 1.25 版本中有几个新的与该功能相关的错误修复，
```
Comment (Michelle951) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:42]:
```suggestion
1.25 版本修复了几个与 CSI 内联存储卷相关的漏洞，
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 10]
```diff
@@ -0,0 +1,226 @@
...
+并且 [CSIInlineVolume 特性门控](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (Michelle951) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:43]:
```suggestion
[并且 [CSIInlineVolume 特性门控](https://kubernetes.io/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 11]
```diff
@@ -0,0 +1,226 @@
...
+已被锁定为 `True`，升级为 GA 通用版。
+因为没有新的 API 变化，所以除了这些错误修复外，beta 版本期间使用该功能的用户应该不会注意到任何重大变化。
```
Comment (Michelle951) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:45]:
```suggestion
因为没有新的 API 变化，所以除了这些错误修复外，使用该功能 beta 版本的用户应该不会注意到任何重大变化。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 12]
```diff
@@ -0,0 +1,226 @@
...
+CSI inline volumes are meant for simple local volumes that should follow the lifecycle of the pod. They may be useful for providing secrets, configuration data, or other special-purpose storage to the pod from a CSI driver.
+
+A CSI driver is not suitable for inline use when:
+- The volume needs to persist longer than the lifecycle of a pod
+- Volume snapshots, cloning, or volume expansion are required
+- The CSI driver requires `volumeAttributes` that should be restricted to an administrator
+-->
+## 何时使用此功能
+
+CSI 内联存储卷是为简单的本地卷准备的，它应该跟随 Pod 的生命周期。
```
Comment (Michelle951) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:62]:
```suggestion
CSI 内联存储卷是为简单的本地卷准备的，这种本地卷应该跟随 Pod 的生命周期。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 13]
```diff
@@ -0,0 +1,226 @@
...
+它们对于从 CSI 驱动向 Pod 提供密码、配置数据或其他特殊用途的存储可能很有用。
```
Comment (Michelle951) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:63]:
```suggestion
它们对于从 CSI 驱动向 Pod 提供密钥、配置数据或其他特殊用途的存储可能很有用。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md (hunk 10 above)
Comment (ldsdsy) [reply content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:43]: Why is there a symbol at the beginning?
Do I still need to keep the "https://kubernetes.io/" before the ”/zh-cn“ ?  I saw a similar suggestion above that was not retained.

**Comment by ldsdsy** [comment]
Thank you both for your review.

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 14]
```diff
@@ -0,0 +1,186 @@
...
+
+<!--
+CSI Inline Volumes were introduced as an alpha feature in Kubernetes 1.15 and have been beta since 1.16. We are happy to announce that this feature has graduated to General Availability (GA) status in Kubernetes 1.25.
+
+CSI Inline Volumes are similar to other ephemeral volume types, such as `configMap`, `downwardAPI` and `secret`. The important difference is that the storage is provided by a CSI driver, which allows the use of ephemeral storage provided by third-party vendors. The volume is defined as part of the pod spec and follows the lifecycle of the pod, meaning the volume is created once the pod is scheduled and destroyed when the pod is destroyed.
+-->
+CSI 内联存储卷是在 Kubernetes 1.15 中作为 Alpha 功能推出的，并从 1.16 开始成为 Beta 版本。
+我们很高兴地宣布，这项功能在 Kubernetes 1.25 版本中已经升级为通用（GA）状态。
+
+CSI 内联存储卷与其他类型的临时卷相似，如  `configMap`、`downwardAPI` 和  `secret`。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:27]:
```suggestion
CSI 内联存储卷与其他类型的临时卷相似，如 `configMap`、`downwardAPI` 和 `secret`。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 15]
```diff
@@ -0,0 +1,186 @@
...
+
+Two existing CSI drivers that support the `Ephemeral` volume lifecycle mode are the Secrets Store CSI Driver and the Cert-Manager CSI Driver.
+
+The [Secrets Store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) allows users to mount secrets from external secret stores into a pod as an inline volume. This can be useful when the secrets are stored in an external managed service or Vault instance.
+
+The [Cert-Manager CSI Driver](https://github.com/cert-manager/csi-driver) works along with [cert-manager](https://cert-manager.io/) to seamlessly request and mount certificate key pairs into a pod. This allows the certificates to be renewed and updated in the application pod automatically.
+-->
+## 使用案例示例
+
+支持`临时`卷生命周期模式的两个现有 CSI 驱动是 Secrets Store CSI 驱动和 Cert-Manager CSI 驱动。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:143]: 1. add a space before and after `` `code` `` syntax.
2. Use `` `Ephemeral` `` or `"临时"` would be better.

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 16]
```diff
@@ -0,0 +1,186 @@
...
+
+[Secrets Store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) 允许用户将 Secret 作为内联卷从外部挂载到一个 Pod 中。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:145]: We can break long lines.
```suggestion
[Secrets Store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver)
允许用户将 Secret 作为内联卷从外部挂载到一个 Pod 中。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md (hunk 15 above)
Comment (ldsdsy) [reply content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:143]: The previous advice in this case was to add spaces in English and no spaces in Chinese. Do you mean that both spaces should be added? So which way does it prevail?

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md (hunk 16 above)
Comment (ldsdsy) [reply content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:145]: Thank you for your advice.

**Comment by ldsdsy** [comment]
@tengqm Please take a look if anything else needs to be changed.

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 17]
```diff
@@ -0,0 +1,187 @@
...
+-->
+**作者：** Jonathan Dobson (Red Hat)
+
+<!--
+CSI Inline Volumes were introduced as an alpha feature in Kubernetes 1.15 and have been beta since 1.16. We are happy to announce that this feature has graduated to General Availability (GA) status in Kubernetes 1.25.
+
+CSI Inline Volumes are similar to other ephemeral volume types, such as `configMap`, `downwardAPI` and `secret`. The important difference is that the storage is provided by a CSI driver, which allows the use of ephemeral storage provided by third-party vendors. The volume is defined as part of the pod spec and follows the lifecycle of the pod, meaning the volume is created once the pod is scheduled and destroyed when the pod is destroyed.
+-->
+CSI 内联存储卷是在 Kubernetes 1.15 中作为 Alpha 功能推出的，并从 1.16 开始成为 Beta 版本。
+我们很高兴地宣布，这项功能在 Kubernetes 1.25 版本中已经升级为通用（GA）状态。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:25]: GA -> 一般可用，正式发布

不要翻译为“通用”

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 18]
```diff
@@ -0,0 +1,187 @@
...
+
+CSI 内联存储卷与其他类型的临时卷相似，如 `configMap`、`downwardAPI` 和 `secret`。
+重要的区别是，存储是由 CSI 驱动提供的，它允许使用第三方供应商提供的临时存储。
+卷被定义为 Pod spec 的一部分，并遵循 Pod 的生命周期，这意味着卷随着 Pod 的调度而创建，并随着 Pod 的销毁而销毁。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:29]:
```suggestion
卷被定义为 Pod 规约的一部分，并遵循 Pod 的生命周期，这意味着卷随着 Pod 的调度而创建，并随着 Pod 的销毁而销毁。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 19]
```diff
@@ -0,0 +1,187 @@
...
+
+A CSI driver is not suitable for inline use when:
+- The volume needs to persist longer than the lifecycle of a pod
+- Volume snapshots, cloning, or volume expansion are required
+- The CSI driver requires `volumeAttributes` that should be restricted to an administrator
+-->
+## 何时使用此功能
+
+CSI 内联存储卷是为简单的本地卷准备的，这种本地卷应该跟随 Pod 的生命周期。
+它们对于从 CSI 驱动向 Pod 提供密钥、配置数据或其他特殊用途的存储可能很有用。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:63]:
```suggestion
它们对于使用 CSI 驱动为 Pod 提供 Secret、配置数据或其他特殊用途的存储可能很有用。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 20]
```diff
@@ -0,0 +1,187 @@
...
+- CSI 驱动需要 `volumeAttributes` 字段，此字段应该限制给管理员使用
+
+<!--
+## How to use this feature
+
+In order to use this feature, the `CSIDriver` spec must explicitly list `Ephemeral` as one of the supported `volumeLifecycleModes`. Here is a simple example from the [Secrets Store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver).
+-->
+## 如何使用此功能
+
+为了使用这个功能，`CSIDriver` spec 必须明确列出 `Ephemeral` 作为 `volumeLifecycleModes` 的参数之一。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:77]:
```suggestion
为了使用这个功能，`CSIDriver` 规约必须明确将 `Ephemeral` 列举为 `volumeLifecycleModes` 的参数之一。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 21]
```diff
@@ -0,0 +1,187 @@
...
+  podInfoOnMount: true
+  attachRequired: false
+  volumeLifecycleModes:
+  - Ephemeral
+```
+
+<!--
+Any pod spec may then reference that CSI driver to create an inline volume, as in this example.
+-->
+然后，任何 Pod spec 都可以引用该 CSI 驱动器来创建一个内联卷，如下例所示。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:95]:
```suggestion
所有 Pod 规约都可以引用该 CSI 驱动来创建一个内联卷，如下例所示。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 22]
```diff
@@ -0,0 +1,187 @@
...
+        driver: secrets-store.csi.k8s.io
+        readOnly: true
+        volumeAttributes:
+          secretProviderClass: "my-provider"
+```
+
+<!--
+If the driver supports any volume attributes, you can provide these as part of the `spec` for the Pod as well:
+-->
+如果驱动程序支持任何卷的属性，你也可以将这些属性作为 Pod `spec` 的一部分。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:123]:
```suggestion
如果驱动程序支持一些卷属性，你也可以将这些属性作为 Pod `spec` 的一部分。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 23]
```diff
@@ -0,0 +1,187 @@
...
+
+Two existing CSI drivers that support the `Ephemeral` volume lifecycle mode are the Secrets Store CSI Driver and the Cert-Manager CSI Driver.
+
+The [Secrets Store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) allows users to mount secrets from external secret stores into a pod as an inline volume. This can be useful when the secrets are stored in an external managed service or Vault instance.
+
+The [Cert-Manager CSI Driver](https://github.com/cert-manager/csi-driver) works along with [cert-manager](https://cert-manager.io/) to seamlessly request and mount certificate key pairs into a pod. This allows the certificates to be renewed and updated in the application pod automatically.
+-->
+## 使用案例示例
+
+支持"临时"卷生命周期模式的两个现有 CSI 驱动是 Secrets Store CSI 驱动和 Cert-Manager CSI 驱动。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:143]:
```suggestion
支持 `Ephemeral` 卷生命周期模式的两个现有 CSI 驱动是 Secrets Store CSI 驱动和 Cert-Manager CSI 驱动。
```

**Comment by tengqm** [comment]
I believe I have reviewed the same blog translation before (https://github.com/kubernetes/website/pull/36880).
It is a dup.

**Comment by ldsdsy** [comment]
Thank you for your review.

**Comment by ldsdsy** [comment]
Is there anything else I should change? I saw that someone translated the same blog post, but I mentioned pr first. Next time I'll write the pr info in more detail in case others didn't see it.

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 24]
```diff
@@ -0,0 +1,187 @@
+---
+layout: blog
+title: "Kubernetes 1.25: CSI 内联存储卷正式发布"
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:3]: Optional:
```suggestion
title: "Kubernetes 1.25：CSI 内联存储卷正式发布"
```

**Review by Sea-n — COMMENTED** [review]
Looks good to me too.
/lgtm

**Pull request merged**